src/main/scala/com/github/kolotaev/ride/Id.scala

```diff
@@ -147,7 +147,7 @@ class Id(bytes: Option[Array[Byte]] = None) extends Serializable with Ordered[Id
   override def equals(other: Any): Boolean = other match {
     case that: Id =>
       (that canEqual this) &&
-        value.deep == that.value.deep
+        value.mkString == that.value.mkString
```

Comment:
@fkoehler, Scala 2.13 removes `ArrayLike#deep`. Are you ok with using `mkString` here? It gives the same effect for 1-dimension byte array.